1.data-exploration/sample_info_data_visualization.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+# import the necessary packages that will be utilized
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd 
+import pathlib
+import seaborn as sns
+
+# assign the desired file a variable using pathlib.Path command
+input_file = pathlib.Path("../0.data-download/data/sample_info.csv")
+
+# set the data frame to be the desired .csv file that is read by pandas(pd) using the pd.read_csv(desired file read as a previously defined variable)
+df = pd.read_csv(input_file)
+
+# print the parameters of the read file
+print(df.shape)
+
+
+# In[2]:
+
+
+# how many samples?
+n_samples = len(df["DepMap_ID"])
+print(f"Number of Samples: {n_samples} \n")
+
+# how many genes 
+all_cancers = df["age"].unique()
+print(f"Ages sampled from: \n {all_cancers} \n")
+
+# how many different types of cancer?
+all_cancers = df["primary_disease"].unique()
+print(f"All Cancer Types: \n {all_cancers} \n")
+
+
+# In[3]:
+
+
+# "Adult" was set to 18
+# "Pediatric" was set to 0
+# "Fetus" was set to 0
+
+# create a new data frame that excludes blank cells in the data sets "Age" column
+no_nan_age_df = df.loc[df["age"].notnull()]
+
+# create a new data frame that will also assign the integer value of 18 to cells containing "Adult"
+no_nan_age_df.loc[(df["age"] == "Adult")] = 18
+
+# create a new data frame that will also assign the integer value of 0 to cells containing "Pediatric"
+no_nan_age_df.loc[(df["age"] == "Pediatric")] = 0
+
+# create a new data frame that will also assign the integer value of 0 to cells containg "Fetus"
+no_nan_age_df.loc[(df["age"] == "Fetus")] = -1
```

Comment:
Oh, I see. You should look around on the DepMap download page for some sort of README documenting details of the file.

-1 usually means _missing data_ in this context.